package.json

```diff
@@ -22,10 +22,12 @@
   "dependencies": {
     "async": "^2.1.5",
     "cntk-fastercnn": "^0.1.2",
+    "crypto-js": "^3.1.9-1",
```

Comment:
For dependencies, I usually lock them instead of having them floating. Totally fine for now -- we can decide if it's a policy we want to adapt going forward. I haven't been burnt often by this, but it definitely happens.